README.md

```diff
@@ -106,7 +106,7 @@ optional arguments:
                         will then fall back to looking for standard files in
                         the current directory that relate to the type of input
                         indicated by the -t flag.
```

Comment:
See comment below, but is the `-t` flag referenced here still valid? (line 108).